package.json

```diff
@@ -11,9 +11,11 @@
   "dependencies": {
     "axios": "^0.19.0",
     "core-js": "^3.3.2",
+    "date-fns": "^2.11.1",
```

Comment:
needless

Reply:
you can use 

import dayjs from 'dayjs'
dayjs(value).format('DD.MM.YYYY')

Reply:
perfectly!
deleted "date-fns". changed date.